database/query/profile_status.sql

```diff
@@ -50,9 +52,9 @@ INSERT INTO rule_details_alert (
 VALUES ($1, $2, $3, sqlc.arg(metadata)::jsonb, NOW())
 ON CONFLICT(rule_eval_id)
     DO UPDATE SET
-                  status = CASE WHEN $2 != 'skipped' THEN $2 ELSE rule_details_alert.status END,
-                  details = CASE WHEN $2 != 'skipped' THEN $3 ELSE rule_details_alert.details END,
-                  metadata = CASE WHEN $2 != 'skipped' THEN sqlc.arg(metadata)::jsonb ELSE rule_details_alert.metadata END,
+                  status = $2,
+                  details = $3,
+                  metadata = sqlc.arg(metadata)::jsonb,
```

Comment:
I think I understand what you are doing now but I can't read the previous code anymore :-) why did we code it like this?

Reply:
Previous comments assume a bunch more context than you might have now:

https://github.com/stacklok/minder/pull/1228/files#r1365307972

Reply:
yeah, I think it was probably wrong to do it like that in the first place. Now it should act the same for remediations and alerts and the logs should reflect the exact state we processed 👍